lib/toml/string.rb

```diff
@@ -16,3 +16,28 @@ def self.transform_escaped_chars(str)
       .gsub(/\\\\/, '\\')
   end
 end
+
+module TomlLiteralString
+  def value
+    first.value[1...-1]
+  end
+end
+
+module TomlMultilineString
+  def value
+    aux = captures[:text].first.value
+
+    # Remove spaces on multilined Singleline strings
+    aux.gsub!(/\\\r?\n[\n\t\r ]*/, '')
+
+    TomlBasicString.transform_escaped_chars aux
```

Comment:
Sadly, `citrus` needs to declare the `value` method at `Module#value` level :/

I guess I can try to do some stuff to refactor it but I prefer to leave it as is.